index.js

```diff
@@ -20,17 +21,72 @@ const releaseOldLockedPortsIntervalMs = 1000 * 15;
 // Lazily create interval on first use
 let interval;
 
-const getAvailablePort = options => new Promise((resolve, reject) => {
-	const server = net.createServer();
-	server.unref();
-	server.on('error', reject);
-	server.listen(options, () => {
-		const {port} = server.address();
-		server.close(() => {
-			resolve(port);
+const getHosts = () => {
+	let interfaces = {};
+	try {
+		interfaces = os.networkInterfaces();
+	} catch (error) {
+		// As of October 2016, Windows Subsystem for Linux (WSL) does not support
+		// the os.networkInterfaces() call and throws instead. For this platform,
+		// assume 0.0.0.0 as the only address
```

Comment:
Please open a Node.js issue about this or link to an existing one.

Reply:
I found https://github.com/nodejs/node/issues/7150, it is not informative bc this is windows related problem

and [window's issue](https://github.com/microsoft/WSL/issues/468#issuecomment-273659271) was closed four years ago, maybe it's time to remove this workaround?

Reply:
> maybe it's time to remove this workaround?

👍

Reply:
Done